.gitignore

```diff
@@ -0,0 +1 @@
+nx/test.ex
```

Comment:
I think you have this leftover from debugging!

Reply:
Sorry, I just realized that. Newbie PR here. I will create a new PR.

Reply:
There is no need, you can just update your fork and it will reflect here :)